tests/e2e/e2e.go

```diff
@@ -51,7 +52,7 @@ type TestEnvironment struct {
 // Retrieve a random URI to naively attempt to spread API load across
 // nodes.
 func (te *TestEnvironment) GetRandomNodeURI() string {
-	r := rand.New(rand.NewSource(time.Now().Unix())) //nolint:gosec
+	r := rand.New(rand.NewSource(time.Now().Unix())) //#nosec G404
```

Comment:
Rather than skipping the whole linter - I think it's better to be explicit on which rule we are silencing.